packages/syft/src/syft/client/client.py

```diff
@@ -653,38 +653,42 @@ def login_as_guest(self) -> Self:
         return _guest_client
 
     def login(
-        self, email: str, password: str, cache: bool = True, register=False, **kwargs
+        self,
+        email: Optional[str] = None,
+        password: Optional[str] = None,
+        cache: bool = True,
+        register: bool = False,
+        **kwargs: Any,
     ) -> Self:
+        if not register and email is None and password is None:
```

Comment:
We've passed guest login to a direct call to `login_as_guest`. Does this affect you in any way?

Reply:
This is just a safeguard in case someone call `login(email=None, password=None)` it would call `login_as_guest`. Of course the canonical way to login as guest is to call `login_as_guest` directly.

Reply:
Yeah, we discussed and `login_as_guest` should be the (explicit) way to login as guest :) login without passing any args should try to populate email and password.

Reply:
Ah ok will change to that.